blog/arquillian-cube-1.13.0.textile

```diff
@@ -0,0 +1,39 @@
+---
+date: 2017-12-18
+author: lordofthejars
+---
+
+p. We are delighted to announce a new version of Arquillian Cube 1.13.0 with new awesome features in Docker part and bug fixes.
+
+p. It is *important* to note that if you are using @env.dependencies@ or @wait.for.service.list@ properties, we have changed the separation char from white-space to comma-separated. To migrate to new version, just add a comma instead of white space in the property to separate each of the element.
```

Comment:
This paragraph sounds a bit disconnected. Maybe start with:

> One of the important fixes we introduced is about handlings multiple values for  @env.dependencies@ and @wait.for.service.list@.  We have changed the separation char from white-space to comma-separated, so to migrate to a new version, just add a comma instead of white space in the property to separate each of the element.

and wrap it in the "important" block. WDYT?